### DISCUSSION TITLE
Any news?

### DISCUSSION THREAD

**Discussion by Giorgi625** [JoomCoder-com/JoomCCK#34] [General]
Hello Andi, do we have any news about approximate release dates of testing versions?
I have project which is done with Cobalt on Joomla 3 and I need to know should I work on promoting it and working with it now and think later about migrating, which maybe painful, or should I wait for JoomCCK and rebuild it on Joomla 4 and then promote.

**Comment by andiamine** [comment]
Hi my friend, currently i don't have an exact date but i can give an estimation , i think maybe from 2 months to 6 months max from now.

as u know Cobalt CCK is a big component with many many features :)

**Comment by andiamine** [comment]
Some screenshots to see the progress
![sc1](https://user-images.githubusercontent.com/6179568/214507232-9b77c7c7-7e0c-4197-ba7e-8ac156dc6ea2.png)
![sc2](https://user-images.githubusercontent.com/6179568/214507234-eba4dc84-0cf8-41e0-a8b6-5067c247076f.png)
![sc3](https://user-images.githubusercontent.com/6179568/214507236-7c237a42-3dec-4c5e-ae7b-86ad7724785a.png)
![sc4](https://user-images.githubusercontent.com/6179568/214507237-33c144dd-4d2f-402e-a117-f60e17950f86.png)

**Comment by Giorgi625** [comment]
It looks nice. Good job.

Can we somehow help in speeding up this process? 

My old clients who have websites on Cobalt are also asking for moving to J4 and php8.1. If I will tell them that it can not be done within 6 monthes, I am afraid they will ask to change extension.

**Comment by andiamine** [comment]
yes some funds can help speedup the update, this will help the team to focus more on JoomCCK.
Also if you want to help on development you are welcome as well.

Currently we work on it everyday but not for many hours we do one to 2 hours per day.

regards,
ANDI

**Comment by Giorgi625** [comment]
I will try to get some funds. 
Can we somehow mention other users, who are following this repository, so they can notice this discussion?

As I mentioned before I can help with frontend templates.

**Reply by andiamine** [reply]
YES you can mention in users :)

**Comment by andiamine** [comment]
beta version will be available next month.

**Comment by Giorgi625** [comment]
Will JoomCCK have same method to get field value like it has Cobalt ? 
Does it make sense to prepare BS5 templates using Cobalt for JoomCCK ?
I use this code
`<?php echo $item->fields_by_key[$params->get('tmpl_params.field_name')]->result; ?>`

**Reply by andiamine** [reply]
Yes same methods and API nothing changed.

**Reply by Giorgi625** [reply]
Great, I will prepare some templates before beta release.

**Reply by andiamine** [reply]
You are welcome.

**Comment by Giorgi625** [comment]
I got notification about tasks that all of them are set as completed.
Does this mean that we will have beta version earlier then expected?

**Comment by andiamine** [comment]
Yes all breaking issues fixed, now we will publish alpha version first before beta so users can help us on testing.
after 2 alpha versions we will publish beta after that stable version (pro and free).

thanks for your patience.
